pytorch_to_returnn/torch/nn/modules/activation.py

```diff
@@ -11,6 +12,17 @@ class _ActivationReturnn(Module):
   def create_returnn_layer_dict(self, input: Tensor) -> Dict[str, Any]:
     return {"class": "activation", "activation": self.func_name, "from": self._get_input_layer_name(input)}
 
+  def _get_output_shape_from_returnn(self, inputs_flat: List[Tensor], layer: LayerBase
+                                     ) -> Tuple[Tuple[int, ...], Dict[int, int]]:
+    """
+    The basic returnn_axis_from_torch_axis should be correct, however, the torch shape is sometimes incorrectly adapted
+    in the base method and we fix it here.
```

Comment:
I don't exactly understand. In what cases? And why?

And this sounds like a bug to me. Is it not? If this is a bug in the base method, we should fix it directly there?

Also, what do you mean "fix it here"? You don't actually do anything here?

Btw, I think there is one case where it might actually change the shape (order of dims), and that is if you use "softmax". Not sure. It might move the feature dim to the end.